charts/prometheus-ipmi-exporter/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if not .Values.configMapFile -}}
```

Comment:
The field `configMapFile` does not occur elsewhere in the chart, I reckon its purpose was somehow not implemented and can therefore safely be removed.